react-components/src/util/TriggerField.js

```diff
@@ -0,0 +1,95 @@
+import SearchField from './SearchField';
```

Comment:
I added this component as well as `SearchField.js` for whenever we eventually convert the entire Data window to React.  I tried to replace `DiskResourceSearchField` with this component, but it currently looks terrible mixed in with the GXT widgets.